spinalcordtoolbox/scripts/sct_register_multimodal.py

```diff
@@ -425,13 +421,16 @@ def main(argv: Sequence[str]):
     elapsed_time = time.time() - start_time
     printv('\nFinished! Elapsed time: ' + str(int(np.round(elapsed_time))) + 's', verbose)
 
-    if path_qc is not None:
-        if fname_dest_seg:
-            generate_qc(fname_src2dest, fname_in2=fname_dest, fname_seg=fname_dest_seg, args=argv,
-                        path_qc=os.path.abspath(path_qc), dataset=qc_dataset, subject=qc_subject,
-                        process='sct_register_multimodal')
-        else:
-            printv('WARNING: Cannot generate QC because it requires destination segmentation.', 1, 'warning')
+    if arguments.qc is not None:
+        qc2.sct_register_multimodal(
```

Comment:
I really like this change.

As you've pointed out, the logic for  _"mapping positional inputs (`path_1`, `path_2`) to keyword inputs (`path_seg`, `path_anat`)"_ is only really needed by the `sct_qc` script, since its interface must be uniform regardless of the given SCT process.

Yet, despite the fact that it's only needed for `sct_qc`, we are currently using this same interface _everywhere_. So, I very much like the idea of confining the "positional input" logic to `sct_qc`, and leaving the "keyword input" for each individual SCT script. It removes the need to scan through a bunch of boilerplate when dealing with QC code outside of `sct_qc`.